app/models/case_contact.rb

```diff
@@ -279,6 +279,12 @@ def self.case_hash_from_cases(cases)
     end
   end
 
+  def form_steps
+    steps = FORM_STEPS.dup
+    steps.delete(:expenses) unless creator.casa_org.additional_expenses_enabled || creator.casa_org.show_driving_reimbursement
```

Comment:
Unless with OR is tough to understand. Can we extract these two into a method that returns true or false?
```
creator.casa_org.additional_expenses_enabled
creator.casa_org.show_driving_reimbursement
```